tests/testthat/test_use_cases.R

```diff
@@ -9,46 +9,48 @@ data <- prostate
 max_formula <- lpsa ~ (. - train)
 model <- regsubsets(max_formula, data=data, nbest = 1, nvmax = 8,
```

Comment:
plz fix: `max_formula` is not found when this is run via `check()`-- do we really need to define a separate variable for the formula? this might work if we supply the formula directly, the indirection here might be the problem when this gets evaluated in the weird testthat-sandbox environment....